src/Module/RemoteFollow.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\DI;
+use Friendica\Core\Logger;
+use Friendica\Core\Protocol;
+use Friendica\Core\Renderer;
+use Friendica\Core\Search;
+use Friendica\Core\System;
+use Friendica\Model\Profile;
+use Friendica\Network\Probe;
+
+/**
+ * Remotely follow the account on this system by the provided account
+ */
+class RemoteFollow extends BaseModule
+{
+	public static function init(array $parameters = [])
+	{
+		if (empty($parameters['profile'])) {
```

Comment:
You don't have to check for the existence/emptiness of `$parameters['profile']`, the router will not select this module for empty values of `{profile}` in the route.